README.md

```diff
@@ -5,8 +5,23 @@ type system, type inference, type providers and a lot of the niceties of OCaml.
 This docker image lets you get started with F# quickly and build applications
 based on F#.
 
-The image contains the latest mono version with F#.
+The `mono` image contains the latest mono version with F#, whereas the `core`
+image contains the latest .NET Core SDK and runtime.
 
 # Dependency versions used:
 
+## Mono Image
 * Mono 4.8.0.495
+
+## .NET Core Image
+* .NET Core 1.1.1 Runtime
```

Comment:
Needs description on 1) how to run a specific executable, 2) how to *compile and run* a specific executable, 3) how to override ENTRYPOINT and how that changes the specifics and 4) what env vars are overridable and what they affect.